install.sh

```diff
@@ -191,6 +191,7 @@ git reset --hard origin/master
 yes | crew install buildessential
 yes | crew install less
 yes | crew install most
+yes | crew install crew_profile
```

Comment:
I would rather this be optional.

Reply:
If this is going to work as I intended, as a total and
complete replacement for your bashrc and zshrc, it will have to be
necessary.

Reply:
A lot of the lines in packages depend on the bashrc. Little, if any, support is given in those packages for zsh and fish. I created this to fix that problem. They can still use their bashrc. It doesn't override it. Same with zshrc and fishrc.

Reply:
You haven't presented your use case example of how this would work.  Are you suggesting all the current packages that have postinstall configuration steps be included in /usr/local/etc/profile instead?  If that is the case, then maybe a `depends_on 'crew_profile'` line should be added in those packages?

Reply:
Yeah. It would just be easier in install. If it's possible, that's what I want. I don't want the zsh and fish users kept in the dark.